parachain/src/lib.rs

```diff
@@ -95,6 +94,12 @@ pub struct ValidationResult {
 #[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize, Debug))]
 pub struct Id(u32);
```

Comment:
suggest move and import this with something like the following as has been done elsewhere in this PR:
```
use primitives::parachain::{
  Id as ThreadId
}
```